src/datachain/lib/dc/datachain.py

```diff
@@ -289,6 +303,13 @@ def session(self) -> Session:
         """Session of the chain."""
         return self._query.session
 
+    @property
```

Comment:
Do we want to use `cached_property` here to do not create new job on a next call?

Reply:
What if two (or more) different sessions will be used in one script?